src/anders_mathiesen_ex/ex01/tidy_code.py

```diff
@@ -0,0 +1,46 @@
+# -*- coding: utf-8 -*-
+
+from random import randint
+
+__author__ = 'Anders Mathiesen'
+__email__ = 'andermat@nmbu.no'
+
+
+def get_guess_bigger_then_or_lik_1():
```

Comment:
please write all code in english. Also i think this function definition is a bit too explicit. could be something like `user_guess()`